test-e2e/test/helpers.js

```diff
@@ -11,10 +9,10 @@ const promiseExec = promisify(childProcess.exec);
 const ESYCOMMAND = require.resolve('../../bin/esy');
 
 async function initFixture(fixture: string) {
-  const rootPath = await fs.mkdtemp('/tmp/esy.XXXX');
+  const rootPath = await fs.mkdtemp('/tmp/esy-test/XXXX');
```

Comment:
All tests are now run in `/tmp/esy-test/...`

Reply:
Unfortunately `/tmp` isn't very friendly on Windows (it will run at the root of the path and may have permissions issues in some build environments) - we should consider using `os.tmpdir()` instead

Reply:
I thought about it wheni wrote it, will change to that 👍